gocourse02/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	zookyper := Zookeeper{}.New(false, "Mukola")
```

Comment:
Помилочка в назві змінної:
```suggestion
	zookeeper := Zookeeper{}.New(false, "Mukola")
```